cloudwatt/test_inter_tenant/README.md

```diff
@@ -0,0 +1,80 @@
+## Usage
+Create terraform.tfvars, it would looks like:
+```
+$ cat terraform.tfvars
+```
+
+```
+auth_url = "XXX"
+region = "XXX"
+
+#credentials tenant A
+name_project_A = "XXXXXX"
+user_project_A = "XXXXXX"
+pwd_project_A = "XXXXXX"
+
+#credentials tenant B:
+name_project_B = "XXXXXX"
+user_project_B = "XXXXXX"
+pwd_project_B = "XXXXXX"
+
+image_name = "XXXXXX"
+flavor_name = "XXXXXX"
+
+#To make it easy, keys would be the same and created before in the 2 tenants
+key_pair_project_A = "XXXXXX"
+key_pair_project_B = "XXXXXX"
+```
+Create the stack:
+```
+$ terraform init
+$ terraform apply -parallelism=1
+```
+
+We can generate hosts file to be used by ansible to play playbook on it using terraform-inventory:
+
+```
+$ terraform-inventory -u <SSH_USER_bastion> -k <SSH_PRIVATE_KEY>
+```
+
+Modify/Check generated file to look like:
+
+```
+[bastion]
+bastion_server_A ansible_host=xx.xx.xx.182 ansible_user=cloud ansible_ssh_private_key_file=/path/private-key ansible_ssh_common_args='-o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no'
```

Comment:
s/cloud/<SSH_USER>/gc
s/\/path\/private-key/<SSH_KEY>/gc

The user will follow that it is the same information like in terrafrom-inventory (line 37)